### ISSUE TITLE
[@kadena/client] Compatibility (layer)

### ISSUE THREAD

**Issue by nillo** [kadena-community/kadena.js#151]
**Is your feature request related to a problem? Please describe.**

**Compatibility:** Kadena client currently does not conform Pact.lang.api
**Functionality** Some enforced functionality should be optional

**suggestion:**
-- un- deprecate pact-lang-api until compatibility layer has been provided  

**Describe the solution you'd like**
Compatibility:

Currently the major wallets make use of the pact-lang-api but kadena client only support signing with chainweaver (quicksign) in order to enforce current project to make use of kadena.js a adaptor should be provided ensure the backwards compatibility of the project. This adaptor is a separate later ensuring the cleanness of kadena client. 

Functionality: 
Kadena client enforces decimals when typeof x === 'number' this is okay for coin.TRANSFER but assumably not for other functions were you might want to provide for example an integer. 

**Additional context**
I will extend this thread with other findings if needed.

**Comment by Randynamic** [comment]
Yes Had a great discussion with the team and until we fixed that part we undeprecate pact-lang-api
